python/sglang/srt/openai_api/adapter.py

```diff
@@ -891,6 +892,7 @@ def v1_chat_generate_request(
                 )
                 if assistant_prefix:
                     prompt_ids += tokenizer_manager.tokenizer.encode(assistant_prefix)
+                prompt = tokenizer_manager.tokenizer.decode(prompt_ids)
```

Comment:
This decode will cause extra overhead, please do not run any extra decode

Reply:
Thank you for the feedback! But this approach seems to be the only viable solution in this case. Do you have any suggestions for alternative implementations?

Reply:
Run tokenizer.apply_chat_template(tokenize=False) first to get the `rendered`, then run `tokenizer(rendered, add_special_tokens=False)` to get the token ids.
See https://github.com/huggingface/transformers/blob/13493215abceafc1653af88b045120014fb4c1fc/src/transformers/tokenization_utils_base.py#L1704-L1737

Reply:
got it. I will fix it.